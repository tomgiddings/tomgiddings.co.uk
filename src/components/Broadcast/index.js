// @flow
import React from 'react';
import Footer from '../Footer'

class Broadcast extends React.Component<*> {
  render() {
    document.title = "Tom Giddings - Broadcast Engineer";
    return (
      <div className="page">
        <div className="content-wrapper broadcast">
          <h1><i className="fas fa-headphones">&nbsp;</i>Broadcast Engineering</h1>
          <p>I've worked as an engineer from Student and Community Radio through to national brands including Absolute Radio, Kiss, Magic, Planet Rock, and Kerrang! Radio.</p>
          <p>In 2017, I became one of the founding directors of <a href="https://www.safepairofhands.co.uk">Safe Pair of Hands Technology</a>, focussing on broadcast technology and IT.</p>
          <ul>
            <li>Studio installation and configuration.</li>
            <li>Worked on Outside Broadcasts including Snowbombing Festival, The Brit Awards, and the Isle of Wight Festival.</li>
            <li>Provided support including out of hours work.</li>
          </ul>
          <Footer />
        </div>
      </div>
    );
  }
}
export default Broadcast;
