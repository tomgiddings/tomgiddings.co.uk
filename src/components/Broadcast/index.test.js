import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Broadcast from './';

afterEach(cleanup);

it('Renders correctly', () => {
  const { asFragment } = render(<Broadcast />);
  expect(asFragment()).toMatchSnapshot();
});
