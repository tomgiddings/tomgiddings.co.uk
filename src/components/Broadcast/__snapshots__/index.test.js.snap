// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<DocumentFragment>
  <article
    class="page broadcast"
  >
    <div
      class="content-wrapper"
    >
      <h1>
        Broadcast Engineer
      </h1>
      <p>
        I've worked as an engineer from Student and Community Radio through to national brands including Absolute Radio, Kiss, Magic, Planet Rock, and Kerrang! Radio.
      </p>
      <p>
        My work has included:
      </p>
      <ul>
        <li>
          Technical installation and refurbishment of broadcast studios.
        </li>
        <li>
          Outside Broadcasts including Snowbombing Festival, The Brit Awards, and the Isle of Wight Festival.
        </li>
        <li>
          Helping to maintain broadcast and IT systems, from studio hardware to VMware vSphere environments.
        </li>
        <li>
          Supporting brands both during and out of hours.
        </li>
      </ul>
      <p>
        I'm currently part of 
        <a
          href="https://www.safepairofhands.co.uk"
        >
          Safe Pair of Hands Technology
        </a>
         working on IT and broadcast projects, and you can find out more about my work on 
        <a
          href="https://www.linkedin.com/in/tomgiddings/"
        >
          <i
            class="fab fa-linkedin"
          >
            Â 
          </i>
          LinkedIn
        </a>
        .
      </p>
      <p>
        <a
          href="https://www.safepairofhands.co.uk"
        >
          <img
            alt="Safe Pair of Hands Technology"
            class="logo"
            src="/logos/spohlogo.png"
          />
        </a>
      </p>
    </div>
  </article>
</DocumentFragment>
`;
